packages/types/src/runtime/transaction-payment/TransactionPayment.ts

```diff
@@ -0,0 +1,15 @@
+import {AnyAssetId} from '@cennznet/types/types';
+
+export function generateTransactionPayment(tip: number, assetId: AnyAssetId, maxPayment: string) {
```

Comment:
We do not actually need this function?  It increases code complexity, and we need to take extra effort on maintaining the relationship between `transactionPayment` and `types` packages. 

When we use it, its not quite clearly on what's meaning of each arguments like `generateTransactionPayment(0, 1, 2);`
Whats more, `types` package is more about typing, and this code segment is kinda data restructuring.

Would suggest to remove this function, and use transactionPayment original data structure directly, like:
```js
const transactionPayment = {
  // tip: tip, // we do not need to assign a value if its value is 0 by default?
  feeExchange: {
    FeeExchangeV1: {
      assetId,
      maxPayment,
    },
  },
};
```

Reply:
@holygits ? 
I don't mind either way... If we can all come to one thing, I can implement that..

Reply:
It's not good to have the construction logic every place we need this type, maybe doing the KV thing helps the readability of the args e.g.: `export function makeTransactionPayment({tip: number, assetId: AnyAssetId, maxPayment: string})`

Reply:
@benxgao - are you okay with a function with argument passed as key-value pair in types package?